winui/Avatar-view/content-types.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# Content Types in WinUI AvatarView
+
+The `AvatarView` control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing the control without the initials or custom image or group source.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView />
+
+{% endhighlight %} 
+{% highlight C# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initials 
+
+The Initials content type is used to display initials in the AvatarView. To use this content type, you must provide a value for the `AvatarName` property, which sets the initials to be displayed in the avatar view. Depending on the specified `InitialsType` property, the initials will be formatted accordingly.
+
+### InitialsType
+
+The `InitialsType` property includes two types:
+
+SingleCharacter - used to display a single character in the avatar.
+DoubleCharacter - used to display two characters in the avatar.
+
+**SingleCharacter**
+
+The single character initials type is used to display the first character of the AvatarName property value as the avatar.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+             
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with SingleCharacter](avatarview_images/winui_single_character_initialstype_avatarview.png)
+
+**DoubleCharacter**
+
+The double character initials type is used to display a two-character text as avatar that has been set in the `AvatarName` property. If the AvatarName consist of a single word, the first and last letters of that word will be displayed. However, if the `AvatarName` contains two or more words, the first letter of the first word and the first letter of the second word will be displayed.
```

Comment:
The double character initials type displays a two-character text as an avatar set in the `AvatarName` property. If the AvatarName consists of a single word, that word's first and last letters will be displayed. However, if the `AvatarName` contains two or more words, the first letter of the first word and the first letter of the second word will be displayed.